tests/conftest.py

```diff
@@ -2683,3 +2684,67 @@ def ceph_toolbox(request):
     if not (deploy or teardown or skip_ocs):
         # Creating toolbox pod
         setup_ceph_toolbox()
+
+
+@pytest.fixture(scope='function')
+def node_drain_teardown(request):
+    """
+    Tear down function after Node drain
+
+    """
+    def finalizer():
+        """
+        Make sure that all cluster's nodes are in 'Ready' state and if not,
+        change them back to 'Ready' state by marking them as schedulable
+
+        """
+        scheduling_disabled_nodes = [
+            n.name for n in get_node_objs() if n.ocp.get_resource_status(
+                n.name
+            ) == constants.NODE_READY_SCHEDULING_DISABLED
+        ]
+        if scheduling_disabled_nodes:
+            schedule_nodes(scheduling_disabled_nodes)
+
+        # Remove label created for DC app pods on all worker nodes
+        node_objs = get_node_objs()
+        for node_obj in node_objs:
+            if 'dc' in node_obj.get().get('metadata').get('labels').keys():
+                remove_label_from_worker_node(
+                    [node_obj.name], label_key="dc"
+                )
```

Comment:
These lines doesn't seem related to node drain tear_down. 
We are removing the label created for DC app pod here.
Any reason for adding this?

Reply:
Nope no reason. Removed